### DISCUSSION TITLE
Timer.freeze();

### DISCUSSION THREAD

**Discussion by SobakaBarabaka74** [Sokol-Games-GmbH/PixelCombats.PublicApi#242] [Ideas]
**Сделать метод freeze в апи таймера.**

Описываю что будет делать данный метод:
* блокировать выполнение таймера. 
* блокировать рестарт таймера, при этом сохраняя уже пройденый интервал времени. 

Например:
```js
function testFunc() {
  msg.Show("Passed!");
  testTimer.freeze();
}

var testTimer = Timers.GetContext().Get("Test");
testTimer.OnTimer.Add(TestFunc);

testTimer.RestartLoop(1); // Результатом выполнения будет Passed 1 раз в msg Show. 
```

По аналогии добавить метод unfreeze();

**Comment by kkohno** [comment]
мысль интересная... вполне можно и реализовать...
